spec/models/appointment_detail_spec.rb

```diff
@@ -0,0 +1,6 @@
+RSpec.describe AppointmentDetail, type: :model do
```

Comment:
```suggestion
RSpec.describe AppointmentDetail do
```

pode remover o type: :model o RSpec já infere o tipo do teste pela pasta que ele ta